File: lang_ts/TaxiCharges.ts
function calculateCharges(distance: number) {
  let charges: number = 0;

  while (distance > 15) {
    charges += 15;
    distance--;
  }

  while (distance > 5) {
    charges += 18;
    distance--;
  }

  while (distance > 2) {
    charges += 20;
    distance--;
  }

  charges += 50;

  return charges;
}

let distance: number = 16;
console.log(
  "Total Chagers For " + distance + "KM.: " + calculateCharges(distance)
);

File: lang_python/TaxiCharges.py
# Taxi Charges

def calculateCharges(distance):

    charges = 0

    while distance > 15:
        charges = charges + 15
        distance = distance - 1

    while distance > 5:
        charges = charges + 18
        distance = distance - 1

    while distance > 2:
        charges = charges + 20
        distance = distance - 1

    charges = charges + 50

    return charges



distance = int(input('Enter distance in KM.: '))
print('Total Charges: ', calculateCharges(distance))